src/main/drivers/serial_uart_impl.h

```diff
@@ -29,53 +29,77 @@
 #define UART_RX_BUFFER_SIZE     128
 #endif
 #ifndef UART_TX_BUFFER_SIZE
+#ifdef USE_MSP_DISPLAYPORT
+#define UART_TX_BUFFER_SIZE     1280
+#else
 #define UART_TX_BUFFER_SIZE     256
 #endif
+#endif
 #elif defined(STM32F3)
 #define UARTDEV_COUNT_MAX 5
 #define UARTHARDWARE_MAX_PINS 4
 #ifndef UART_RX_BUFFER_SIZE
 #define UART_RX_BUFFER_SIZE     128
 #endif
 #ifndef UART_TX_BUFFER_SIZE
+#ifdef USE_MSP_DISPLAYPORT
+#define UART_TX_BUFFER_SIZE     1280
```

Comment:
Have you tested building a F3 target with this?

Reply:
no

Reply:
I tried but i'm failed.
```` ./src/main/drivers/bus_spi.c:250:2: error: #error "Base SPI clock not defined for this architecture" ````
````250 | #error "Base SPI clock not defined for this architecture"````

Reply:
My point is that this should not be changed for MCU types that are deprecated and that this has not been tested on.

Reply:
I will remove it.